src/engine/graph_node.h

```diff
@@ -132,9 +132,11 @@ struct GraphNodeOrthoProjection {
 struct GraphNodePerspective {
     /*0x00*/ struct FnGraphNode fnNode;
     /*0x18*/ s32 unused;
-    /*0x1C*/ f32 fov;   // horizontal field of view in degrees
-    /*0x20*/ u16 near;  // near clipping plane
-    /*0x22*/ u16 far;   // far clipping plane
+    /*0x1C*/ f32 halfFovHorizontal;
+    /*0x20*/ f32 halfFovVertical;
+    /*0x24*/ f32 fov;   // horizontal field of view in degrees
+    /*0x26*/ u16 near;  // near clipping plane
```

Comment:
This offset is not correct. Also is there actually any reason for us to leave the `unused` field in this struct?